src/libcrun/container.c

```diff
@@ -2504,9 +2504,21 @@ libcrun_container_exec (libcrun_context_t *context, const char *id, runtime_spec
         }
 
       if (context->kontain) {
-        int rc = libcrun_kontain_argv(&process->args, &exec_path);
+        char* new_execpath;
+        int rc = libcrun_kontain_nonkmexec_allowed(exec_path, &new_execpath);
         if (rc != 0) {
-          libcrun_fail_with_error (rc, "exec: fixup argv");
+          libcrun_fail_with_error(rc, "allow non-km executable check failed");
```

Comment:
does it simply mean "Failed to parse config file xxx"  ?

Reply:
It could mean failed to parse the config file, or some error from a system call.
Ideally the message should come from the called function and it probably does.
This is just the final place we check for an error and issue a generic message.
The fact that crun's error/warning messages are at times hard to find results in having these redundant messages.
If I believed that crun error messages would appear someplace where a person could see them I wouldn't have included this message.